lib/puppet/face/node/clean.rb

```diff
@@ -52,7 +52,7 @@ def debug(message)
     end
 
     def warn(message)
-      Puppet.warning(message)
+      Puppet.warning(message) unless message =~ /cadir is currently configured to be inside/
```

Comment:
@puppetlabs/puppetserver-maintainers , is this change expected or should we keep the `main` version?